### DISCUSSION TITLE
new races

### DISCUSSION THREAD

**Discussion by BASON73** [War3Evo/War3Source-EVO#19] [Q&A]
Are there any new races for this mod, or where can I find them?

**Comment by War3Evo** [comment]
I'm sure there are thousands of races out there.  The only issue will be converting them for War3Source: EVO as it's changed alot from the original. 

Also know, not all races are compatible with the different games (TF2, CSGO, CSS, FOF).

You could be the first person to request others to build races for you in [Discussions General Topic area](https://github.com/War3Evo/War3Source-EVO/discussions/categories/general)

As of right now, I'm going thru all the current game servers to hunt down bugs and fix them.  When the bugs get all fixed, I may have time to develop more races.
